doc/source/networking.rst

```diff
@@ -164,8 +164,11 @@ Be sure the default gateway knows the route back to the VNET subnets.
 
 **Using VLANs**
 
-If using VLAN interfaces for the jail host, add the VLAN interface AND
-parent interface of the VLAN as bridge members.
+To assign a jail's traffic to a VLAN, add the VLAN interface as a bridge
+member, but not the VLAN's parent.
+
+If using VLAN interfaces for the jail host, on the other hand, add the parent
+as a bridge member, but not the VLAN interface.
```

Comment:
It's been a while since I've messed with vlans and jails. This was originally written by @pannon , but I believe both the vlan interface and the parent interface need to be added, not just the parent. But I could be wrong.

Reply:
As @nathanjrobertson and myself have both found, adding the parent interface to the bridge is harmful.  The VLANed jail can only pass traffic when the vlan's parent is _not_ in the bridge.